src/main/java/catan/domain/transfer/output/game/GameUserDetails.java

```diff
@@ -26,6 +30,7 @@ public GameUserDetails(GameUserBean gameUserBean, int detailsRequesterId) {
         this.achievements = new AchievementsDetails(gameUserBean.getAchievements());
 
         if (user.getId() == detailsRequesterId) {
+            this.availableActions = GSON.fromJson(gameUserBean.getAvailableActions(), AllAvailableActionsDetails.class);
```

Comment:
btw, why should we convert JSON to Java class and after that convert Java class to JSON? is it possible to return Json stored in Java calss that will be converted to Json?